synapse/config/sso.py

```diff
@@ -110,6 +114,14 @@ def generate_config_section(self, **kwargs):
             #client_whitelist:
             #  - https://riot.im/develop
             #  - https://my.custom.client/
```

Comment:
```suggestion
            #  - https://my.custom.client/

```

Heads up options should have a blank link separating them from each other for readability. See the [configuration file format guide](https://matrix-org.github.io/synapse/develop/code_style.html#configuration-file-format) for more details.